src/libraries/System.Net.Security/src/System/Net/Security/SecureChannel.cs

```diff
@@ -806,7 +806,7 @@ private SecurityStatusPal GenerateToken(byte[] input, int offset, int count, ref
                         status = SslStreamPal.AcceptSecurityContext(
                                       ref _credentialsHandle,
                                       ref _securityContext,
-                                      input != null ? new ArraySegment<byte>(input, offset, count) : default,
+                                      input, offset, count,
```

Comment:
Why replace ArraySegment with three parameters? If it's changing, going to Span/ROSpan seems better.